applications/lookerbot/main.tf

```diff
@@ -0,0 +1,30 @@
+# This template builds an S3 bucket for a Lookerbot instance.
+#
+# See https://github.com/looker/lookerbot#amazon-s3
+#
+# Manual setup steps:
+#   - Finally, head to the S3 Bucket in AWS after provisioning, and go to the
```

Comment:
It'd be nice to outline the manual steps for setting this up here, e.g:

```
- Head to <https://git.io/fh9DS> and hit "Deploy to Heroku" to provision the app.
- Set the required environment variables via Heroku's admin panel. <https://git.io/fh9Dy>
- Set the `SLACKBOT_S3_BUCKET`, `AWS_ACCESS_KEY_ID` & `AWS_SECRET_ACCESS_KEY`
  environment variables from the resources we've provisioned in this module.
```

We also probably don't want to lock down public access here, since this is used for file hosting.

Reply:
Ah, good point. We can remove that S3 instruction entirely, can't we?

Reply:
Yep!